coldfront/core/utils/management/commands/audit_data.py

```diff
@@ -0,0 +1,313 @@
+            # options['lrc_all'] = flag_enabled('LRC_ONLY')
+        if options.get('lrc_all', False):
+            options['lrc_user_billing'] = True
+            options['lrc_recharge_allocation_billing'] = True
+            options['lrc_recharge_allocation_user_billing'] = True
+            options['lrc_pis_lbl_employee'] = True
+        for option in options:
+            if isinstance(options[option], bool) and options[option] and \
+                    option not in ('all', 'lrc_all'):
+                self.stdout.write(self.style.SUCCESS('Running {}...' \
+                    .format(option)))
+                getattr(self, 'handle_{}'.format(option))()
+                self.stdout.write(self.style.SUCCESS('Audit complete.') + '\n\n')
+
+    def handle_allocation_date(self):
+        '''
+        Assert that all compute allocations have the expected start_date
+        and end_date values.
+        - Allocation end date is after the start date
+        - FCAs and PCAs:
+          FCAs and PCAs are valid within a so-called “Allowance Year” 
+          (June 1st - May 31st for BRC, October 1st - September 30th for LRC):
+            - Inactive ones (ones that were not renewed) should have a start
+                date set to the start of the current allowance year
+                AllocationPeriod, but no end date.
+            - Active ones (ones that were renewed, or were created during
+                this allowance year) should have the same start and end date as
+                the current allowance year AllocationPeriod.
+          ICAs (Instructional Computing Allowances) are valid within a
+          particular UC Berkeley semester.
+            - Inactive ones should have a start date, but no end date.
+            - Active ones (currently none on production) should match a current
+                (has started, but has not ended) instructional AllocationPeriod.
+          Recharge and Condo may/should have start dates, but don’t have end
+          dates, as they don’t end.
+        '''
+        allocations = Allocation.objects.select_related('project',
+                            'project__status').prefetch_related('resources') \
+            .filter(resources__name__endswith='Compute') \
+            .order_by('resources__name', 'project__name',
+                      'project__status__name', 'start_date', 'end_date') \
+            .values('id', 'project__status__name',
+                   'project__name','start_date', 'end_date',
+                   'resources__name')
+
+        FCA_PCA_ALLOCATION_PERIOD = AllocationPeriod.objects.get(
+                                        name__startswith='Allowance Year',
+                                        start_date__lt=datetime.date.today(),
+                                        end_date__gt=datetime.date.today())
+        ICA_ALLOCATION_PERIODS = AllocationPeriod.objects.filter(
+                                    Q(end_date__gt=datetime.date.today())
```

Comment:
You should use `oldfront.core.utils.common.display_time_zone_current_date` to get a date that takes the time zone into account.